testassets/Shared/InteropClient.cs

```diff
@@ -168,7 +168,7 @@ private static WinHttpHandler CreateWinHttpHandler()
         {
 #pragma warning disable CA1416 // Validate platform compatibility
             var handler = new WinHttpHandler();
-            handler.SslProtocols = SslProtocols.Tls12 | SslProtocols.Tls11 | SslProtocols.Tls;
+            handler.SslProtocols = SslProtocols.Tls12;
```

Comment:
what's the impact of this change?

Reply:
This is test only. Removed because older TLS enum values are obsolete.